README.md

```diff
@@ -12,14 +12,14 @@
 [![Maintainability](https://api.codeclimate.com/v1/badges/9161568d8378cea903f4/maintainability)](https://codeclimate.com/github/trailofbits/manticore/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/9161568d8378cea903f4/test_coverage)](https://codeclimate.com/github/trailofbits/manticore/test_coverage)
 
-Manticore is a symbolic execution tool for analysis of binaries and smart contracts.
+Manticore is a symbolic execution tool for analysis of smart contracts and binaries.
```

Comment:
Since we now more fully support macOS, do we want to specify "Linux binaries"?

Reply:
i'd prefer to keep it as is, i think that makes this line look a little cluttered, and i think it's ok without